maint/GenerateUcd.py

```diff
@@ -980,6 +998,6 @@ def write_bitsets(list, item_size):
 /* End of pcre2_ucd.c */
 """)
 
-f.close
+f.close()
```

Comment:
this is unnecessary and might had been intentional (I think i'd seen more of those as well), so maybe better done independently (if needed)